job_definitions/notify_suppliers_of_awarded_briefs.yml

```diff
@@ -37,16 +41,22 @@
     builders:
       - shell: |
           if [ -n "$AWARDED_AT" ]; then
-            FLAGS="--awarded_at=$AWARDED_AT"
+            FLAGS="--awarded-at=$AWARDED_AT"
           fi
 
           if [ -n "$BRIEF_RESPONSE_IDS" ]; then
-            FLAGS="$FLAGS --brief_response_ids=$BRIEF_RESPONSE_IDS"
+            FLAGS="$FLAGS --brief-response-ids=$BRIEF_RESPONSE_IDS"
+          fi
+
+          if [ -n "$RESUME_BUILD_NUMBER" ]; then
+            FLAGS="$FLAGS --reference=${JOB_NAME}_${RESUME_BUILD_NUMBER}"
+          else
+            FLAGS="$FLAGS --reference=${JOB_NAME}_${BUILD_NUMBER}"
           fi
 
           if [ "$DRY_RUN" = "true" ]; then
             FLAGS="$FLAGS --dry-run"
           fi
 
-          docker run --rm -e DM_DATA_API_TOKEN_{{ environment|upper }} digitalmarketplace/scripts scripts/notify-suppliers-of-awarded-briefs.py '{{ environment }}' $NOTIFY_API_TOKEN {{ NOTIFY_TEMPLATE_ID }} $FLAGS --verbose
+          docker run --mount type=bind,source="$(pwd)",target=/tmp --rm -e DM_DATA_API_TOKEN_{{ environment|upper }} digitalmarketplace/scripts scripts/notify-suppliers-of-awarded-briefs.py --stage '{{ environment }}' --notify-api-key $NOTIFY_API_TOKEN --notify-template-id {{ NOTIFY_TEMPLATE_ID }} $FLAGS --verbose
```

Comment:
Is this binding to the magic Jenkins job workspace directory which gets cleaned up every so often?

Reply:
@alex9smith yes, `$(pwd)` in this instance is the Jenkins workspace directory which is where the script is run from and persists after the test run.